lib/DensityLib.sol

```diff
@@ -116,32 +116,33 @@ library DensityLib {
   }}
 
   // Convenience function: get a fixed-point density from the given parameters. Computes the price of gas in outbound base tokens, then multiplies by cover_factor.
```

Comment:
I think the meaning is "base as opposed to display units" (since one argument is in `outbound_display_...`). Maybe `in wei of outbound tokens` or `in base units of outbound tokens`?

Reply:
Ah, I see. Not sure what the good solution is. IMO the ambiguity of using `wei` et al. for other things than native is problematic and should be avoided. So `base units` is probably better.

Reply:
addressed here a7e62d0d3da31bcc78957f9b825c3361fd9240cd